ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/sdn/OWNERS

```diff
@@ -1,14 +1,18 @@
 approvers:
 - jianlinliu
+- Amoghrd
 - yunjiang29
-- gpei
+- mgahagan73
+- MayXuQQ
```

Comment:
Are those expected changes in this PR ?

Reply:
I just feel confuse why this file be changed, I will rebase it.